### PR TITLE
Some reformat code for 1.11.2 source

### PR DESCRIPTION
**Pull request by KorDum**
First iteration.
Deleted autogen IDEA comment from each file.
Optimized imports.
Rearrange class members.
_Code style like 1.7.10_

### CODE REVIEW

File: src/main/java/binnie/botany/genetics/ModuleGenetics.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 	@Override
 	public void preInit() {
-		/* INIT API*/
+	    /* INIT API*/
```
Comment (mezz) [inline src/main/java/binnie/botany/genetics/ModuleGenetics.java]: I think this was right before

File: src/main/java/binnie/core/proxy/BinnieModProxy.java [hunk 2]
```diff
@@ -103,10 +103,10 @@ public void registerModel(ModelEntry index) {
 	}
 
 
-//	@Override
-//	public IIcon getIcon(final IIconRegister register, final String string) {
-//		return BinnieCore.proxy.getIcon(register, this.mod.getModID(), string);
-//	}
+	//	@Override
```
Comment (mezz) [inline src/main/java/binnie/core/proxy/BinnieModProxy.java]: You should try to avoid indenting all the commented-out everywhere.

**Review by mezz — CHANGES_REQUESTED** [review]

File: src/main/java/binnie/core/proxy/BinnieModProxy.java (hunk 2 above)
Comment (KorDum) [reply src/main/java/binnie/core/proxy/BinnieModProxy.java]: Fixed all comments

File: src/main/java/binnie/botany/genetics/ModuleGenetics.java (hunk 1 above)
Comment (KorDum) [reply src/main/java/binnie/botany/genetics/ModuleGenetics.java]: Hmm. Autoformat mistake.

**Pull request merged**